Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
 import './App.css';
+import CalcRow from './components/CalcRow/CalcRow';
+import DisplayRow from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
+  const total = 0;
+  const row5 = [{symbol:'clear',type:'act',color:'white',size:'cell75'},{symbol:'%',type:'act',color:'red',size:'cell25'}]
+  const row4 = [{symbol:'7',type:'num',color:'white',size:'cell25',bold:'bold'},{symbol:'8',type:'num',color:'white',size:'cell25',bold:'bold'},{symbol:'9',type:'num',color:'white',size:'cell25',bold:'bold'},{symbol:'X',type:'act',color:'red',size:'cell25'}]
```

Comment:
way to really go to town on the props!

However, at a certain point I would say let the component do this presentational work for you, at a certain point you're making the component more customizable than than you need to, and the resulting code isn't very DRY. In the real world, you could just create a separate component to deal with an edge case like 0, or you would use defaultProps